src/include/sof/drivers/idc.h

```diff
@@ -93,6 +93,10 @@
 #define IDC_MSG_PREPARE_D0ix		IDC_TYPE(0x9)
 #define IDC_MSG_PREPARE_D0ix_EXT	IDC_EXTENSION(0x0)
 
+/** \brief IDC process async msg */
+#define IDC_MSG_AMS	IDC_TYPE(0xA)
+#define IDC_MSG_AMS_EXT	IDC_TYPE(0x0)
```

Comment:
what is the difference between plain vanilla AMS and AMS_EXT?

Put differently, why do we need two options?